hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/file/FileSourceBuilder.java

```diff
@@ -163,6 +189,13 @@ public String path() {
         return format;
     }
 
+    /**
+     * Returns if the filesystem is shared. Only valid for local filesystem, distributed filesystems are always shared.
+     */
+    public boolean sharedFileSystem() {
```

Comment:
Boolean getters tend to be confusing:

```java
FileSourceBuilder<String> source = FileSources.files("src/test/resources/directory/")
                                              .withFormat(FileFormat.lines());
source.sharedFileSystem();
```

Unless there's a smart IDE warning that warns you're ignoring the boolean return value (you're ignoring it all the time anyway, with `return this` methods), this will be a very surprising thing.

A better name is`isSharedFileSystem`.